### PR TITLE
Csv movie

### PR DESCRIPTION
**Pull request by oandrevictor**

### CODE REVIEW

File: client/views/movie.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+      <div class="uk-width-xxlarge">
+        <div class="title">{{movie.name}}
+          <span ng-show="movie.isBoxOffice" class="uk-badge purple">In teathers</span>
+        </div>
+
+      </div>
+    </div>
+    <div class="uk-width-xxlarge content">
+      <div class="uk-width-3-4">
+              <b>Directed by</b> {{ tvshow.directors }}
```
Comment (gabrielmla) [inline client/views/movie.html]: movie.directors

File: client/views/movie.html [hunk 2]
```diff
@@ -0,0 +1,60 @@
+<div class="movie">
+    <!--Cover Content-->
+    <div class="uk-background-cover" align="center">
+      <div class="uk-width-xxlarge">
+        <div class="title">{{movie.name}}
+          <span ng-show="movie.isBoxOffice" class="uk-badge purple">In teathers</span>
+        </div>
+
```
Comment (DavidMedeiros) [inline client/views/movie.html]: remove this line (8)

File: client/views/movie.html [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+      </div>
+    </div>
+    <div class="uk-width-xxlarge content">
+      <div class="uk-width-3-4">
+              <b>Directed by</b> {{ tvshow.directors }}
+              <br /> Year:
```
Comment (DavidMedeiros) [inline client/views/movie.html]: **...br/>** instead **...br />**

File: client/views/movie.html [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+              <b> {{ release_date_formated }}</b> | Genres:
```
Comment (DavidMedeiros) [inline client/views/movie.html]: remove blank space before "{{"

**Pull request merged**